.github/workflows/pythonpackage.yml

```diff
@@ -11,6 +11,9 @@ on:
       - "**.lock"
       - "Makefile"
 
+env:
```

Comment:
This adds colour to our output on GitHub Actions, so we can see if any prior test has failed at a glance.